### PR TITLE
Update docs for scaleup and upgrade

### PR DESCRIPTION
**Pull request by mtnbikenc**
https://jira.coreos.com/browse/CORS-1054

### CODE REVIEW

File: HOOKS.md [hunk 1]
```diff
@@ -3,20 +3,16 @@
 OpenShift Ansible allows for operators to execute custom tasks during
 specific operations through a system called hooks. Hooks allow operators to
 provide files defining tasks to execute before and/or after specific areas
-during installations and upgrades. This can be very helpful to validate
-or modify custom infrastructure when installing/upgrading OpenShift.
+during upgrades. This can be very helpful to validate
+or modify custom infrastructure when upgrading OpenShift.
```
Comment (vrutkovs) [inline HOOKS.md]:
```suggestion
or modify custom infrastructure when upgrading RHEL nodes in OpenShift.
```

**Comment by mtnbikenc** [comment]
Need to fix the missing pem in CI before this will pass.
/hold

**Comment by vrutkovs** [comment]
/test e2e-aws-scaleup-rhel7

https://github.com/openshift/release/pull/3365 should have fixed the missing file in the secret

File: README.md [hunk 2]
```diff
@@ -15,54 +15,55 @@ Installation of OpenShift 4.x uses a command-line installation wizard instead of
...
-mycluster-worker-2.example.com
+Required variables include:
+
+- `openshift_kubeconfig_path` - Path to the kubeconfig for the cluster
+- `openshift_pull_secret_path` - Path to the pull secret to the image registry
+
+## Run the RHEL node scaleup playbook
+
+```bash
+ansible-playbook -i ~/inventory/hosts playbooks/scaleup.yml
```
Comment (patrickdillon) [inline README.md]: should `~/inventory/hosts` be `inventory/hosts`?

File: README.md [hunk 3]
```diff
@@ -15,54 +15,55 @@ Installation of OpenShift 4.x uses a command-line installation wizard instead of
...
 ```
 
-## Run the scaleup playbook
+## Run the RHEL node upgrade playbook
+Custom tasks can be performed during upgrades at different stages of the upgrade.
+See the [hooks documentation](HOOKS.md) for more information.
 
 ```bash
-ansible-playbook playbooks/openshift_node/scaleup.yml
+ansible-playbook -i ~/inventory/hosts playbooks/upgrade.yml
```
Comment (patrickdillon) [inline README.md]: same as above, should `~/inventory/hosts` be `inventory/hosts`?

File: README.md (hunk 2 above)
Comment (mtnbikenc) [reply README.md]: Updated.

File: README.md (hunk 3 above)
Comment (mtnbikenc) [reply README.md]: Updated.

**Comment by mtnbikenc** [comment]
Docs only PR
/override ci/prow/e2e-aws-scaleup-rhel7

**Pull request merged**